From a67e2ea630f01086ef390a12827d398bcfccdfd0 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Mon, 15 Sep 2014 14:07:52 +0800
Subject: [PATCH] ENGR00331450-6: ARM: dts: imx6-sabresd: add dts for ldo
 enable mode

Add dts for ldo enable mode on imx6q/dl-sabresd board.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/Makefile               |  2 ++
 arch/arm/boot/dts/imx6dl-sabresd-ldo.dts | 19 +++++++++++++++++++
 arch/arm/boot/dts/imx6q-sabresd-ldo.dts  | 10 ++++++++++
 3 files changed, 31 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 8094ced..7a25bad 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -159,6 +159,7 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx6dl-sabreauto-flexcan1.dtb \
 	imx6dl-sabresd.dtb \
 	imx6dl-sabresd-enetirq.dtb \
+	imx6dl-sabresd-ldo.dtb \
 	imx6dl-sabresd-hdcp.dtb \
 	imx6dl-wandboard.dtb \
 	imx6q-arm2.dtb \
@@ -168,6 +169,7 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx6q-sabreauto-flexcan1.dtb \
 	imx6q-sabrelite.dtb \
 	imx6q-sabresd.dtb \
+	imx6q-sabresd-ldo.dtb \
 	imx6q-sabresd-enetirq.dtb \
 	imx6q-sabresd-uart.dtb \
 	imx6q-sabresd-hdcp.dtb \
diff --git a/arch/arm/boot/dts/imx6dl-sabresd-ldo.dts b/arch/arm/boot/dts/imx6dl-sabresd-ldo.dts
new file mode 100644
index 0000000..b24f2fe
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dl-sabresd-ldo.dts
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2014 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6dl-sabresd.dts"
+
+&cpu0 {
+	arm-supply = <&reg_arm>;
+	soc-supply = <&reg_soc>;
+};
+
+&gpc {
+	/* use ldo-enable, u-boot will check it and configure */
+	fsl,ldo-bypass = <0>;
+};
diff --git a/arch/arm/boot/dts/imx6q-sabresd-ldo.dts b/arch/arm/boot/dts/imx6q-sabresd-ldo.dts
index 6dda35c..99ea580 100644
--- a/arch/arm/boot/dts/imx6q-sabresd-ldo.dts
+++ b/arch/arm/boot/dts/imx6q-sabresd-ldo.dts
@@ -7,3 +7,13 @@
  */
 
 #include "imx6q-sabresd.dts"
+
+&cpu0 {
+	arm-supply = <&reg_arm>;
+	soc-supply = <&reg_soc>;
+};
+
+&gpc {
+	/* use ldo-enable, u-boot will check it and configure */
+	fsl,ldo-bypass = <0>;
+};
-- 
1.8.0

